fleet-secrets-bro-patch/README.md

```diff
@@ -0,0 +1,26 @@
+# Fleet | GitRepo Secret Backup Restore Patch
+
+This is a patching script to ensure all secrets used by Fleet `GitRepos` are backed up by the Racnher Backups tool.
+
+The script is designed to fix existing clusters using Fleet and Rancher Backups. When run, it ensures that all secrets used by GitRepos are backed up. Although a bug fix is in progress, it will not be retroactive. Therefore, this script can resolve the current issue.
```

Comment:
```
The script is designed to fix existing clusters using Fleet and Rancher Backups. When run, it ensures that all secrets used by GitRepos are backed up. Although a bug fix is in progress, it will not be retroactive. Therefore, this script can resolve the current issue.

Additionally, the bug fix will only address `Secrets` created via the Fleet UI of Rancher. It does not cover secrets created outside of the Fleet UI, as those secrets cannot be guaranteed to be "fleet-owned."
```

Suggestion (to be updated with actual versions)

```
From Rancher v2.8.?? and v2.9.0 all `Secrets` created via the Fleet UI in Rancher will be included in Rancher Backups. 

Any GitRepo `Secrets` created before this, or outside of the Fleet UI in Rancher, will not be included in Rancher Backups. 

By running this patching script ....
```

Reply:
Great improvement. I'll make sure to come back and edit the 2.8 tag once we know for sure what version that will be. Since users/customers might benefit from this fix sooner than those are released.